policyengine_us/variables/gov/states/ma/tax/income/credits/ma_dependent_or_dependent_care_credit.py

```diff
@@ -13,9 +13,12 @@ class ma_dependent_or_dependent_care_credit(Variable):
     defined_for = StateCode.MA
 
     def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.states.ma.tax.income.credits.dependent_care
```

Comment:
Is the child and family tax credit a completely new credit concept or did MA just merge the two previous credits? I dont think I see the exact definition in the 2021 tax forms

Reply:
The question is whether the rules stay the same or if we need to re-write the logic